lib/lint-messages.js

```diff
@@ -0,0 +1,66 @@
+/** Lint messages, find missing messages and those that are unused */
+
+const YAML = require('yamljs')
+const { readFileSync } = require('fs')
+const glob = require('glob')
+
+const IMPORT = /import ([a-zA-Z0-9]+) from '[^ ]+\/messages'/
+
+module.exports = function () {
+  // TODO don't hardwire paths
+  const messages = YAML.parse(readFileSync('configurations/default/messages.yml', 'utf8'))
```

Comment:
for the location of the `yml` files, you should use the config sent in other places of mastarm.  See for example:  https://github.com/conveyal/mastarm/blob/dev/bin/mastarm#L24